DZ_2.py

```diff
@@ -0,0 +1,40 @@
+#list()
+a = [10, 11, 2, 3, 5, 8, 23, 11, 2, 5, 76, 43, 2, 32, 76, 3, 10, 0, 1]
+
+
+
+d = sorted(a)
+print(d[-3], d[-2], d[-1]) # 1.2 in homework
+
+
+
+print(a.index(min(a))) # 1.3 in homework
+
+b = list(reversed([10, 11, 2, 3, 5, 8, 23, 11, 2, 5, 76, 43, 2, 32, 76, 3, 10, 0, 1]))
+print(b)  # 1.4 in homework
+
+
+for i in reversed(range(len(a))):
+    if a[i] in a[:i]:
+        del a[i]
+print(a) # 1.1 in homework
+print(set(a)) # 1.1 in homework если нужно отобразить масив без повторений и удалений
```

Comment:
set(a) return set, but not list. So, you need to convert the set to a list `list(set(a))`